tests/ffi/common/README.md

```diff
@@ -0,0 +1,21 @@
+Title: OStree images using podman pull to pull containers, larger than 1G.
```

Comment:
I suggest to move this readme to https://github.com/containers/qm/tree/main/docs
You can create directory here like qm containers.conf with README


please look simple explanation, explain how to change the variables  in qm containers.conf
And a link to the section in tests for sample code.

Reply:
> I suggest to move this readme to https://github.com/containers/qm/tree/main/docs You can create directory here like qm containers.conf with README
> 

Done.

> please look simple explanation, explain how to change the variables in qm containers.conf And a link to the section in tests for sample code.

I've made changes to explanations in README, if I've got you right.